Source/Core/Common/ChunkFile.h

```diff
@@ -200,6 +200,14 @@ class PointerWrap
     DoArray(arr, static_cast<u32>(N));
   }
 
+  void* DoExternal(u32& count)
```

Comment:
```suggestion
  [[nodiscard]] void* DoExternal(u32& count)
```

Reply:
This should also return `u8*`, rather than hiding the type